README.md

```diff
@@ -70,6 +70,26 @@ Used for optimizing images that come from backdrops and emblems.
  * `IMAGE_OPTIMIZE_KEY` - The key for API use.
  * `IMAGE_DOMAIN` - The domain for API Service
 
+### Docker
+A docker-based infrastructure is available for development.
+
+Set the database hostname in `.env`
+```
+DB_HOST=leaf-db
+```
+
+To set up the containers, run
+```shell
+docker compose --env-file .env -p leaf -f docker/docker-compose.yml up -d
```

Comment:
This didn't work for my until I swapped `docker compose` for `docker-compose` due to "unknown flag --env-file"

Reply:
This must be your docker version being old. Idk why it would be out-of-sync with your docker-compose (unless they were out of parity at some point). Do you want me to change it to old docker-compose, or we can just require a certain docker version?

Reply:
Unsure. If others have the up to date, that seems fine. I'll keep changing though since I believe mine is up to date against apt.

```
➜  Leaf git:() ✗ docker-compose --version
Docker Compose version v2.2.3
➜  Leaf git:(r) ✗ docker --version        
Docker version 20.10.22+dfsg1, build 3a2c30b
``